packages/@react-types/cards/src/index.d.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps, DOMProps, StyleProps} from '@react-types/shared';
+import {ReactNode} from 'react';
+
+interface AriaCardProps extends AriaLabelingProps {}
+
+interface SpectrumCardProps extends AriaCardProps, StyleProps, DOMProps {
+  children: ReactNode,
+  isQuiet?: boolean,
+  layout?: 'grid' | 'waterfall' | 'gallery',
+  size?: 'S' | 'M' | 'L',
```

Comment:
just a heads up but commented out this prop in my pull since we don't currently support it